sys/include/net/gnrc/pkt.h

```diff
@@ -109,18 +119,18 @@ typedef struct gnrc_pktsnip {
     /* the first three fields *MUST* match iolist_t! */
     struct gnrc_pktsnip *next;      /**< next snip in the packet */
     void *data;                     /**< pointer to the data of the snip */
-    size_t size;                    /**< the length of the snip in byte */
+    gnrc_pkt_size_t size;           /**< the length of the snip in byte */
```

Comment:
iolist_t needs to also be adjusted

Reply:
uh but why?

Reply:
because it is casted to `iolist_t *` when passed to netdevs

Reply:
:fearful: 
then that's a can of worms that I rather leave the lid on